torchtitan/config_manager.py

```diff
@@ -220,6 +220,7 @@ def __init__(self):
         self.parser.add_argument(
             "--optimizer.fused",
             action="store_true",
+            default=True,
             help="Whether the fused implementation(CUDA only) is used.",
```

Comment:
`action="store_true"` and `default=True` will make it impossible to turn it off from cmd line.
I'd recommend switching to the following
```suggestion
            "--optimizer.disable_fused",
            action="store_true",
            help="Whether the fused implementation(CUDA only) is used.",
```

Reply:
Thanks for the feedback.  You can turn it off by using [optimizer] fused=false in the toml but I see what you mean re: command line.
I'll add this.